conjure-java-core/src/integrationInput/java/com/palantir/product/EteService.java

```diff
@@ -77,6 +77,12 @@ String path(
             @HeaderParam("Authorization") @NotNull AuthHeader authHeader,
             @PathParam("param") String param);
 
+    @GET
+    @Path("base/externalLong/{param}")
+    String externalLongPath(
```

Comment:
Missing an implementation in EteResource.java

Reply:
Shoot! Will add that in! Also going to change it to return Long as well. Accidentally put string.